persistence/persistence-mybatis/src/main/resources/org/cbioportal/persistence/mybatis/GeneMapper.xml

```diff
@@ -14,6 +14,17 @@
             gene.LENGTH AS "${prefix}length"
         </if>
     </sql>
+
+    <sql id="selectRefGene">
+        gene.ENTREZ_GENE_ID AS "${prefix}entrezGeneId",
+        gene.HUGO_GENE_SYMBOL AS "${prefix}hugoGeneSymbol"
+        <if test="projection == 'SUMMARY' || projection == 'DETAILED'">
+            ,
+            gene.TYPE AS "${prefix}type",
+            reference_genome_gene.cytoband AS "${prefix}cytoband",
```

Comment:
there is no join with this `reference_genome_gene` table?

Reply:
it will be supplied by calling mutationMapper object.

Reply:
when? In next PR?

Reply:
@pieterlukasse , sorry missed this one. In this PR, if you take a look at MatationMapper.xml file you will find join statements.